### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by bharatkram**

### CODE REVIEW

File: project1/templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <head>
+        {% styling %}{% end %}
+        <title>{{ headline }}</title>
+        <style>
+        </style>
+    </head>
+    <body>
+        <h1>{% heading %}{% end %}</h1>
+        {% body %}{% end %}
+    </body>
```
Comment (madhulika9293) [inline project1/templates/layout.html]: good job with the layout!

File: project1/application.py [hunk 2]
```diff
@@ -24,3 +24,16 @@
 @app.route("/")
 def index():
     return "Project 1: TODO"
```
Comment (madhulika9293) [inline project1/application.py]: Add a welcome page instead of sending text.

**Comment by madhulika9293** [comment]
Make the change and you can merge into the master.

File: project1/application.py (hunk 2 above)
Comment (bharatkram) [reply project1/application.py]: i have now added a page index.html.
Is it ok now?

**Pull request merged**